prepline_general/api/general.py

```diff
@@ -169,7 +169,12 @@ def partition_pdf_splits(
     pages_per_pdf = int(os.environ.get("UNSTRUCTURED_PARALLEL_MODE_SPLIT_SIZE", 1))
 
     # If it's small enough, just process locally
+    # (Some kwargs need to be renamed for local partition)
     if len(pdf_pages) <= pages_per_pdf:
+        if "hi_res_model_name" in partition_kwargs:
+            partition_kwargs["model_name"] = partition_kwargs["hi_res_model_name"]
```

Comment:
do we want to rename `hi_res_model_name` to `model_name` for consistency? (why we have two different names two

Reply:
That's a good question - might have been to make it more clear when the param was relevant. I think we'll want to revisit all the params soon. We've maybe outgrown the api-tools pattern of making everything a list. We'll need to keep the old name for backwards compat, but this would be a good chance to make those consistency fixes.